### ISSUE TITLE
 Dose ElastAlert 0.2.0b support Elasticsearch 6.3.0 and Kibana 6.3.0?

### ISSUE THREAD

**Issue by leee66** [Yelp/elastalert#2403]
I will plan to use ElastAlert 0.2.0b. The version of my Elasticsearch and Kibana is 6.3.0. Dose ElastAlert 0.2.0b support Elasticsearch 6.3.0 and Kibana 6.3.0?

**Comment by wildgarden** [comment]
No, it dose not.

**Comment by leee66** [comment]
> No, it dose not.

Which version of ElastAlert does support Elasticsearch 6.3.0 and Kibana 6.3.0?

**Comment by abhishekjiitr** [comment]
ElastAlert 0.1.39 should work fine with ES v6.X

**Comment by leee66** [comment]
> ElastAlert 0.1.39 should work fine with ES v6.X

Thank you.

**Comment by Qmando** [comment]
@wildgarden 
Why do you say it doesn't? Are you getting errors?

**Comment by leee66** [comment]
> @wildgarden
> Why do you say it doesn't? Are you getting errors?

If I setup ElastAlert 0.2.1, when 【pip install "elasticsearch<=6.3.0"】 is executed, get the following error:
ERROR: elastalert 0.2.1 has requirement elasticsearch>=7.0.0, but you'll have elasticsearch 6.3.0 which is incompatible.

**Comment by Qmando** [comment]
Hm, I forgot the exact circumstances in which setuptools will actually prevent you from downgrading a package, though I know there's a way around it. You get that error DURING the pip install??

You can definitely get around this by downloading from source, editing requirements.txt and re-installing it (python setup.py install).

**Comment by leee66** [comment]
> Hm, I forgot the exact circumstances in which setuptools will actually prevent you from downgrading a package, though I know there's a way around it. You get that error DURING the pip install??
> 
> You can definitely get around this by downloading from source, editing requirements.txt and re-installing it (python setup.py install).

Yes, I got that error during 【pip install "elasticsearch<=6.3.0"】.
Where is requirements.txt?
